specs/binding-tcp.spec/src/main/resources/META-INF/zilla/tcp.idl

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+scope tcp
+{
+    scope event
+    {
+        enum TcpEventType (uint8)
+        {
+            REMOTE_ACCESS_FAILED (1)
```

Comment:
Looking at the acceptance criteria for #755 , the TCP event we need is DNS resolution failed (unknown host exception).

Reply:
In the tcp binding this network error is caught here:
https://github.com/aklivity/zilla/blob/c3b8844a1b3caacae89d50e8f89313250512561d/runtime/binding-tcp/src/main/java/io/aklivity/zilla/runtime/binding/tcp/internal/stream/TcpClientFactory.java#L289
The scope looks more general so I was thinking I give this event a more generic name as we might want to log other types of errors here as well. Maybe we can check the exception class and pick various error/event types similar to the tls case. WDYT? 🤔❓

Reply:
OK